sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGenerateExec.scala

```diff
@@ -367,7 +371,7 @@ abstract class GpuExplodeBase extends GpuUnevaluableUnaryExpression with GpuGene
     val inputRows = inputBatch.numRows()
 
     // if the number of input rows is 1 or less, cannot split
-    if (inputRows <= 1) return Array()
+    //if (inputRows <= 1) return Array()
```

Comment:
I would love to see us push `inputSplitIndices` into the individual generate expressions. But that can be follow on as it really is just tech debt.

Also why is the row commented out?

Reply:
Nice catch, that should not be commented out.

I'll look into the inputSplitIndices refactor.

Reply:
Filed this for the refactor of `inputSplitIndices` and more... https://github.com/NVIDIA/spark-rapids/issues/10187. I don't want to block this issue further.